### DISCUSSION TITLE
not able to use my back camera for the poof

### DISCUSSION THREAD

**Discussion by TheAbyssSlayer** [community/community#118691] [GitHub Education]
### Hi everyone,

I tried to use my back camera so I can click a picture on my phone but the picture screen went black

**Comment by samus-aran** [comment]
Hi @TheAbyssSlayer 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

> 1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
> 2. If you can, print off your evidence and take a photo of this copy.
> 3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
> 4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
> 5. Alternatively you may use another mobile device or computer.

Hope this helps 👍

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)
